browser/src/Editor/NeovimEditor.tsx

```diff
@@ -176,7 +175,9 @@ export class NeovimEditor extends Editor implements IEditor {
         this._neovimInstance.on("event", (eventName: string, evt: any) => {
             const current = evt.current || evt
             this._updateWindow(current)
-            this._buf = this._bufferManager.updateBufferFromEvent(current)
+            if (eventName !== "bufEnter") {
```

Comment:
Hmm, does this check work? I thought it needed to be `BufEnter`

Reply:
I've pushed a typo fix for that but ended up removing it having had a look at the function as the if statement there seems to be of little value as logging it, that function gets called very frequently and is not very expensive i.e. does not result in any async actions etc. on top of which it gets called by the `bufWinEnter`event then subsequently by `bufEnter`event then `bufRead` then `cursorsMoved` etc so preventing it being called one extra time seemed to have little value, ideally separating it into the essential bits for buffer updating and a function that returned the desired `OniBuffer` would be ideal as it stands i have to call it in the `bufEnter` handler to get the return  as well as in the general event handler

Reply:
Ah ok, makes sense!